### PR TITLE
Remove global ia components

### PR DESCRIPTION
**Pull request by caseysinclair**

### CODE REVIEW

File: components/Layout/Layout.module.scss [hunk 1]
```diff
@@ -7,7 +7,9 @@
 .navigationBar {
   @extend %navigation-bar;
   box-sizing: border-box;
-  width: 60px;
+  position: sticky;
```
Comment (rodleviton) [inline components/Layout/Layout.module.scss]: @caseysinclair I don't think has any support in IE11?

File: components/Layout/_styles.scss [hunk 2]
```diff
@@ -83,16 +78,15 @@
...
-  overflow-x: hidden;
-  overflow-y: auto;
-  position: fixed;
-  width: $sidebar-width;
-  z-index: $ca-z-index-fixed;
-  padding: $ca-grid;
+  width: $ca-layout-sidebar-width;
+  padding: $ca-grid 0;
+  overflow-x: visible;
+  overflow-y: visible;
```
Comment (rodleviton) [inline components/Layout/_styles.scss]: would `overflow: visible` do the same thing?

File: components/NavigationBar/NavigationBar.js [hunk 3]
```diff
@@ -92,36 +115,25 @@ export default class NavigationBar extends React.Component<Props, State> {
...
-  menuChange = (open: boolean) => {
-    this.setState(state => ({
-      ...state,
-      menusOpen: state.menusOpen + (open ? 1 : -1),
-    }));
-  };
-
   static displayName = 'NavigationBar';
-
   static defaultProps = {
```
Comment (rodleviton) [inline components/NavigationBar/NavigationBar.js]: According to the type definition all of these props are required. If we are giving default values should we make those props optional?

File: components/NavigationBar/components/Link.js [hunk 4]
```diff
@@ -4,56 +4,93 @@ import classNames from 'classnames';
...
-        {...{ href, id, onClick }}
-      >
-        <Icon icon={icon} title={tooltip} />
-      </a>
-      <div
-        className={classNames(styles.tooltip, {
-          [styles.suppressed]: hideTooltip,
+        className={classNames(styles.link, {
+          [styles.active]: active,
+          [styles.containsText]: typeof text != 'undefined',
```
Comment (rodleviton) [inline components/NavigationBar/components/Link.js]: What is the reason for this kind of check? Also is not using the strict comparison operator e.g. `!==`.

File: components/NavigationBar/components/Link.module.scss [hunk 5]
```diff
@@ -1,44 +1,169 @@
...
+  margin: ($ca-grid / 4) 0;
 
   // center child
   align-items: center;
   justify-content: center;
 
   border-radius: $ca-border-radius;
 
-  color: #fff;
-  composes: interactiveIconWrapper from 'cultureamp-style-guide/components/Icon/Icon.module.scss';
```
Comment (rodleviton) [inline components/NavigationBar/components/Link.module.scss]: This is a win. No longer dependent on styles from Icon component.

File: components/NavigationBar/components/Link.module.scss [hunk 6]
```diff
@@ -1,44 +1,169 @@
...
+        .linkIcon {
+          opacity: 1;
+        }
+      }
+    }
   }
 }
 
-.active {
-  composes: active from 'cultureamp-style-guide/components/Icon/Icon.module.scss';
```
Comment (rodleviton) [inline components/NavigationBar/components/Link.module.scss]: 🎉

**Comment by rodleviton** [comment]
[Migrate new NavigationBar and Layout to Kaizen Component Library](https://trello.com/c/kzpvOwKx/163-migrate-new-navigationbar-and-layout-to-kaizen-component-library)

File: components/Layout/Layout.module.scss (hunk 1 above)
Comment (caseysinclair) [reply components/Layout/Layout.module.scss]: Good pickup!

**Pull request merged**